app/code/Magento/Catalog/view/frontend/templates/product/image_with_borders.phtml

```diff
@@ -12,7 +12,7 @@ $width = (int)$block->getWidth();
 $paddingBottom = $block->getRatio() * 100;
 ?>
 <span class="product-image-container product-image-container-<?= /* @noEscape */ $block->getProductId() ?>">
```

Comment:
We probably don't need this ID any more given the other changes in this pull request.
```suggestion
<span class="product-image-container">
```